### ISSUE TITLE
训练样本

### ISSUE THREAD

**Issue by SorryMaker666** [sjtuplayer/anomalydiffusion#36]
您好，在您的论文中提到We employ one-third of the anomaly data with the lowest ID numbers as the training set, reserving the remaining two-thirds for testing，想请问您一下，这里的the anomaly data是使用了test文件夹和ground_truth文件夹下的数据吗，因为我看到train文件夹下的数据都是good无缺陷的数据

**Comment by SorryMaker666** [comment]
> @SorryMaker666 MVTec AD provides the test set for defects, so in anomaly diffusion they split the test set into two parts.

好的，谢谢你的回答！！
同时我还有一个问题，在运行时是否同时出现
/root/miniconda3/envs/Anomalydiffusion/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py:1566: UserWarning: GPU available but not used. Set the gpus flag in your trainer `Trainer(gpus=1)` or script `--gpus=1`
和RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument index in method wrapper__index_select)
这两个报错，我在两台电脑上运行都会出现这个问题，请问这个该怎么解决
